csrc/neighbor_sample.cpp

```diff
@@ -52,10 +52,10 @@ hetero_temporal_neighbor_sample(
     const c10::Dict<node_t, torch::Tensor> &input_node_dict,
     const c10::Dict<rel_t, std::vector<int64_t>> &num_neighbors_dict,
     const c10::Dict<node_t, torch::Tensor> &node_time_dict,
-    const int64_t num_hops, const bool replace, const bool directed) {
+    const int64_t num_hops, const bool replace) {
```

Comment:
Let's keep the interface as it is. This way we do not need to make any modifications to PyG.

Reply:
done